db.json

```diff
@@ -1,3 +1,43 @@
 {
-  "events": []
-}
+  "events": [
+    {
+      "eventName": "My trip to Hawaii",
+      "eventLocation": "Hawaii, HI",
+      "eventDescription": "Hello there\r\n",
+      "startDate": {
```

Comment:
I recommend storing these as normal date strings. You can use the `fecha` package for formatting.